codalab/lib/codalab_manager.py

```diff
@@ -497,6 +517,69 @@ def client(self, address, is_cli=True):
     def cli_verbose(self):
         return self.config.get('cli', {}).get('verbose')
 
+    def get_access_token(self, address, auth_handler):
```

Comment:
The terrible part is when client is a RemoteBundleClient:
RemoteBundleClient.login() makes an RPC call to the LocalBundleClient.login() on the server, to get the token from the auth handler on the server. I wasn't sure how to redesign this without wasting too much time on code that will be defunct soon anyway.